sections/dashboard/FuturesMarketsTable/FuturesMarketsTable.tsx

```diff
@@ -436,6 +437,15 @@ const StyledText = styled.div`
 	font-family: ${(props) => props.theme.fonts.bold};
 `;
 
+const ColoredPrice = styled.div<{ color: string }>`
+	color: ${(props) =>
+		props.color === 'red'
+			? props.theme.colors.selectedTheme.red
+			: props.color === 'green'
+			? props.theme.colors.selectedTheme.red
```

Comment:
is `props.color === 'green' ? props.theme.colors.selectedTheme.red` meant to be `props.theme.colors.selectedTheme.red` 

And if we had the prop `color: PriceColor ` we could just do `props.theme.colors.selectedTheme[props.color]`

Reply:
This type doesn't exactly work because we pass it through the data in the table. I opted to just use `'white` | 'red' | 'green'` here even though we create this type elsewhere.